tethysapp/hydro_var_monitor/templates/hydro_var_monitor/home_language_components/en/admin.html

```diff
@@ -0,0 +1,120 @@
+{% extends "hydro_var_monitor/base.html" %}
+
+{% load static %}
+
+{% block header_buttons %}
+    <!-- <button  id="instructions" type="button" class="btn btn-primary btn-block" data-bs-toggle="modal" data-bs-target="#myModal" ><i class="bi bi-info-circle"></i></button> -->
+    <div class="header-button" data-bs-toggle="tooltip" data-bs-placement="bottom" title="Instructions">
+        <a data-bs-toggle="modal" data-bs-target="#myModal">
+            <i class="bi bi-info-circle-fill"></i>
+        </a>
+    </div>
+{% endblock %}
+
+{% block app_navigation_items %}
+   <li class="nav-item">
+        <a class="nav-link" href="{% url 'hydro_var_monitor:admin' %}">Admin</a>
+    <li class="nav-item">
+        <a class="nav-link" href="{% url 'hydro_var_monitor:home' %}">Home</a>
+    </li>
+
+{% endblock %}
+
+{% block app_content %}
+    <h2>Step 1: Is EE Enabled?</h2>
+    <div>EE Enabled?: {{ ee_enabled }}</div>
+    <div>
+        If Earth Engine isn't enabled on this portal with a service account,
+        <a href="https://developers.google.com/earth-engine/service_account" target="_blank">
+            click here to learn how to do this.</a>
+    </div>
+    <!--break-->
+    </br><br>
+
+    <h2>Step 2: Upload Polygons</h2>
+    <div class="form-group">
+        <p> Upload 2 zip files for your desired regions. Inside the zip files should be all the json
+            files you would like included in the app. The title of each of the files should have the
+            name you would like included on the dropdown menu to choose from. The two zip files
+            should include the exact same names for each of the files.</p>
+        <p>This first zipfile should contain json files with the exact boundaries. All the points
+            can be included in this shapefile. This file will show how the regions appear on the map in the
+            app.</p>
+        <label for="exact-json" class="col-form-label">Exact JSON:</label>
+        <input type="file" class="form-control" id="exact-json">
+    </div>
+    </br><br>
+    <div class="form-group">
+        <p>This second zipfile should contain json files with the approximate boundaries. The json files must be the
+            exact same names as there corresponding json file. If there are
```

Comment:
add a link to download an example zipped dataset with a single json in the exact and simplified directories for reference.

Reply:
put it in the public directory, perhaps in a subfolder called samples or examples or something like that.